clients/desktop/src/chain/tx/components/TxOverviewMemo.tsx

```
@@ -1,12 +1,49 @@
+  const [state, setState] = useState(initialState)
+  const { parsedMemo } = state
+
+  const componentDidMount = (): void => {
+    getParsedMemo(value)
+      .then(parsedMemo => {
+        setState(prevState => ({ ...prevState, parsedMemo }))
+      })
+      .catch(() => {})
+  }
+
+  useEffect(componentDidMount, [])
```

ESLint

[error] React Hook useEffect has a missing dependency: 'value'. Either include it or remove the dependency array.

(react-hooks/exhaustive-deps)